tests/conftest.py

```diff
@@ -592,3 +594,9 @@ def closed_orders(billing_address):
     orders.append(Order.objects.create(billing_address=billing_address))
 
     return orders
+
+
+@pytest.fixture
+def variant_choice_field_form():
```

Comment:
I think this doesn't have to be a fixture. We use fixtures mostly for reusable data needed in various tests.